build.xml

```diff
@@ -1330,6 +1345,7 @@ fi</echo>
 		<echo message="tests.dir.rel=${tests.dir.rel}" if:set="debug.true"/>
 		<echo message="tests.dirset.includes=${tests.dirset.includes}" if:set="debug.true"/>
 		<echo message="tests.properties:${tests.properties}" if:set="debug.true"/>
+		<echo message="h2.jar:${h2.jar}" if:set="debug.true"/>
```

Comment:
This line is already present at line 1328

Reply:
Ik heb hem weggehaald.